apps/org/views.py

```diff
@@ -56,3 +66,213 @@ def get_queryset(self):
         """A user may only delete Organizations that they are associated with."""
         qs = super().get_queryset()
         return qs.filter(users=self.request.user)
+
+
+@login_required(login_url='home')
+def org_create_member_view(request, org_slug):
+    """A view for allowing a User at an Organization to create a Member for that Organization."""
+    organization = get_object_or_404(
+        Organization.objects.filter(users=request.user),
+        slug=org_slug
+    )
+
+    # If the user is GETting the page, then render the template.
+    if request.method == 'GET':
```

Comment:
Move this logic to a class based view if possible, so that the `get` vs `post` logic can be in separate methods.